collections/_posts/2024-12-02-SSL_comparison.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: review
+title: "Systematic comparison of semi-supervised and self-supervised learning for medical image classification"
+tags: classification, semi-supervision, self-supervision
+author: "Juliette Moreau"
+cite:
+    authors: "Zhe Huang, Ruije Jiang, Shuchin Aeron and Michael C. Hughes"
+    title:   "Systematic comparison of semi-supervised and self-supervised learning for medical image classification"
+    venue:   "CVPR, 2024"
+pdf: "https://arxiv.org/pdf/2307.08919"
+---
+
+
+# Highlights
+
+* Benchmark of SSL methods on medical images.
+* Implementation of a real life hyperparameters optimization.
+* Comparison of semi-supervised and self-supervised learning.
+
+# Note
+
+Code is available here: https://github.com/tufts-ml/SSL-vs-SSL-benchmark
+
+# Introduction
+
+In medical domain there is often a lack of annotated data for deep learning model training while there is a lot of unlabeled data in data health records. Semi-supervised and self-supervised learning are methods developed to take advantage of these unlabeled images to improve classification accuracy. The first one train classifiers jointly with two loss terms, while the second is a two-stages approach, the first to learn deep representation and the second to fine-tune the classifier. However, the two methods are rarely compared. The first question they try to answer is: **Which recent semi- or self-supervised methods are likely to be most effective?** \
```

Comment:
- In **the** medical domain
- "while there is a lot of unlabeled data in data health records" → Awkward phrasing. Suggested: "while a large amount of unlabeled data exists in health records."
- The first one **trains** classifiers jointly
- the second is a **two-stage** approach
- General comment : maybe you could gain in lisibility with bullet points for Semi and SSL such as : 

- Semi : blablabla introductive content
- SSL: introductive content with two-stage approach

1. Pretraining
2. Finetuning


Suggestion of rephrasing of chatGPT (2ème review pour le prix d'une hihi): 
In the medical domain, annotated data for deep learning is often scarce, while a large amount of unlabeled data exists in health records. Semi-supervised and self-supervised learning methods have been developed to leverage this unlabeled data and improve classification accuracy. The former trains classifiers jointly with two loss terms, while the latter follows a two-stage approach: first, learning deep representations, and second, fine-tuning the classifier. However, these two methods are rarely compared.